pnpm-lock.yaml

```diff
@@ -3,8 +3,10 @@ lockfileVersion: 5.3
 specifiers:
   '@babel/core': ^7.13.15
   '@babel/eslint-parser': ^7.13.14
+  '@types/react-datepicker': ^4.1.7
```

Comment:
Try reverting this lock file as it seems you removed theses dependencies in root without using `pnpm` thus breaking the lock-file and build 🙈